src/test/java/org/opensearch/search/asynchronous/restIT/AsynchronousSearchRestTestCase.java

```diff
@@ -86,15 +86,29 @@ public void indexDocuments() throws IOException {
         client().performRequest(new Request(HttpPost.METHOD_NAME, "/_refresh"));
     }
 
-    AsynchronousSearchResponse executeGetAsynchronousSearch(GetAsynchronousSearchRequest getAsynchronousSearchRequest) throws IOException {
```

Comment:
Can we revert this if it's an unintended indentation?

Reply:
no the code has changed a bit as i've overloaded this method